docs/Reference/Reply.md

```diff
@@ -499,7 +499,7 @@ reply
   .serializeInput({ foo: 'bar'}, 200) // '{"foo":"bar"}'
 ```
 
-See [.compileSerializationSchema(schema, httpStatus)](#compileserializationschema)
+See [.compileSerializationSchema(schema, [httpStatus])](#compileserializationschema)
```

Comment:
why this change?

Reply:
As per previous [comment](https://github.com/fastify/fastify/pull/4139#discussion_r922661183)